plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptionsVariationSelectorAttributeName.php

```diff
@@ -23,4 +25,41 @@ class AddToCartWithOptionsVariationSelectorAttributeName extends AbstractBlock {
 	protected function get_block_type_style() {
 		return null;
 	}
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 * @return string Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ): string {
+		global $attribute_name;
+
+		$content = '';
+
+		if ( isset( $attribute_name ) ) {
+			$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes, array(), array( 'extra_classes' ) );
+	
+			$wrapper_attributes = get_block_wrapper_attributes(
+				array(
+					'class' => esc_attr( $classes_and_styles['classes'] ),
+					'for'   => esc_attr( 'attribute_' . sanitize_title( $attribute_name ) ),
+					'id'    => esc_attr( 'attribute_' . sanitize_title( $attribute_name ) . '_label' ),
```

Comment:
Wondering if we should prefix the ids with the product slug, so if there are several Single Products blocks with different products using the same attributes we don't end up with duplicate ids.